src/createButton.ts

```diff
@@ -12,11 +14,26 @@ export interface ButtonFactory extends ComposeFactory<Button, ButtonOptions> { }
 
 const createButton: ButtonFactory = createWidget
 	.mixin({
-		mixin: createFormFieldMixin
+		mixin: createFormFieldMixin,
+		aspectAdvice: {
+			before: {
+				getNodeAttributes(this: Button, overrides: VNodeProperties = {}) {
+					if (this.state.classes) {
+						const classes: { [index: string]: boolean; } = {};
+						this.state.classes.forEach((c) => classes[c] = true);
+						classes[ css.button ] = true;
```

Comment:
Need to make it more explicit in our style guide, but array literals are spaced, but index properties are not, so `classes[css.button] = true`